x11-wm/pekwm/pekwm-0.2.1.ebuild

```
@@ -0,0 +1,87 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib desktop
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+S="${WORKDIR}"/${PN}\-${PV}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="contrib debug +jpeg +png themes truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libXrender
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	themes? ( dev-vcs/git )"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DENABLE_XSHAPE="yes"
+		-DENABLE_XRANDR="yes"
+		-DENABLE_JPEG=$(usex jpeg)
+		-DENABLE_PNG=$(usex png)
+		-DENABLE_XINERAMA=$(usex xinerama)
+		-DENABLE_XFT=$(usex truetype)
+		-DENABLE_XPM=$(usex xpm)
+	)
+
+	cmake_src_configure
+}
+
+multilib_src_install() {
+
+	cmake_src_install
+
+	# Install contributor scripts into doc folder
+	if use contrib; then
+		docinto contrib
+		dodoc "${S}"/contrib/lobo/*.{pl,vars,png} "${S}"/contrib/lobo/README
+	fi
+
+	# Insert an Xsession
+	echo -e "#!/bin/sh\n\n/usr/bin/${PN}" > "${T}"/"${PN}" || die
+	exeinto /etc/X11/Sessions
+	doexe "${T}"/"${PN}"
```

```suggestion
exeinto /etc/X11/Sessions
newexe - pekwm <<- _EOF_
	#!/bin/sh
	/usr/bin/pekwm
_EOF_
```
should also work. Note I didn't test.

---

I tested this, it works.